optimum/intel/openvino/trainer.py

```diff
@@ -168,7 +168,8 @@ def __init__(
             model_inputs = next(iter(train_dataloader))
             for label_name in self.label_names:
                 model_inputs.pop(label_name)
-            self.ov_config.add_input_info(model_inputs)
+            force_single_batch = self._should_apply_pruning_transform()
```

Comment:
@AlexKoff88 @echarlaix Made the fix more local. Now it affects only pruning transform case.
Similar what is implemented for ngraph export here:
https://github.com/huggingface/optimum-intel/blob/3714c3226da20a78d2fb1008da2e3592f294b7f1/optimum/intel/openvino/trainer.py#L707-L710